### DISCUSSION TITLE
Duckstation

### DISCUSSION THREAD

**Discussion by Blue03147** [community/community#28198] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I installed duckstation on an old computer. When running the game, I encountered an error (failed to create host display device context). (failed to acquire host display) Can it be fixed?

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're trying to report a bug in a software hosted on GitHub, maybe https://github.com/stenzek/duckstation? In that case you should open an issue *in that repository*. I assume the developer(s) there would appreciate a bit more context if you do. :slightly_smiling_face:

**Reply by Blue03147** [reply]
How?

**Reply by airtower-luna** [reply]
Peculiar, the owner seems to have disabled issues on the repository. That's something they can do, but really unusual, sorry I didn't notice! The README doesn't mention any bug reporting/support ways either. The owner seems to have a public mail address on their profile and the README lists a Discord channel, so I guess you could try whichever of those seems more comfortable to you. :sweat_smile: 

My main point is that this space is primarily about Git and GitHub. If you post about bugs in random software here there's little chance the developers will see your report – basically none unless they happen to hang around here.

**Reply by Blue03147** [reply]
Thanks

**Comment by queenofcorgis** [comment]
👋🏼 It looks like you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

1. Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
2. Check the [SUPPORT file](https://help.github.com/articles/adding-support-resources-to-your-project/), if one exists, for instructions on how to best contact the maintainers for support.
3. Check the [CONTRIBUTING guide](https://help.github.com/articles/setting-guidelines-for-repository-contributors/), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by airtower-luna** [reply]
That's basically what I said already! :sweat_smile:

**Reply by queenofcorgis** [reply]
@airtower-luna you're right - have to give credit where credit is due 😄

**Reply by airtower-luna** [reply]
Thank you! :smile_cat:
